addons/core/translations/resources/en-us.yaml

```diff
@@ -827,10 +827,15 @@ credential-store:
   messages:
     welcome:
       title: Welcome to Credential Stores
+    no-filters:
+      title: No worker filters added
+      description: You haven't added any worker filters yet
   titles:
     new: New Credential Store
+    worker-filter: Worker Filter
   actions:
     delete: Delete Credential Store
+    add: Add worker filters
```

Comment:
Well I think this description exists bc there are two types of credential stores (vault, static) but for this feature, it is only available for vault credentials

Reply:
I could reuse `title` but that specific text for `description` isn't really what we need in this instance

Reply:
I also think title is diff in this case bc it refers to filter singular

Reply:
The translations @ZedLi linked are used for targets. What we could do is refactor the worker filter related translations since we have more than one resource that uses it. I wouldn't make that a blocker for this PR and I can make a ticket to address this in a different PR.

Reply:
Link to [ticket](https://hashicorp.atlassian.net/browse/ICU-15794)